google-cloud-pubsub/README.md

```diff
@@ -43,13 +43,35 @@ subscriber = sub.listen do |received_message|
   received_message.acknowledge!
 end
 
+# Handle exceptions from listener
+subscriber.on_error do |exception|
+  puts "Exception: #{exception.class} #{exception.message}"
+end
+
 # Start background threads that will call the block passed to listen.
 subscriber.start
 
 # Shut down the subscriber when ready to stop receiving messages.
 subscriber.stop.wait!
 ```
 
+This example will drain the Pub/Sub queue and then exit.  If you wish to block and listen to messages as they are published, block after the call to `#start` and be sure to handle teardown.  For example:
+
+```ruby
+subscriber = sub.listen do |received_message|
```

Comment:
It's a requirement of this repo that all examples are complete. Since this example is separate from the one above, you would need to copy lines 24-38 to make it complete.

**HOWEVER**, it's also a convention in this repo to show just a single "headline" example in the README.md, with additional examples in OVERVIEW.md. I think the best solution here is to merge all of your additions into the single example.